src/nupic/algorithms/Cells4.hpp

```diff
@@ -406,8 +407,8 @@ namespace nupic {
                Real permInc =.1,
                Real globalDecay =0,
                bool doPooling =false,
-               int seed =-1,
-               bool doItAll =false,
+               int seed =42,
+               bool initFromCpp =true,
```

Comment:
Can we move this change of default value to separate PR (with the seed as well)?